packages/table/src/TableHeader.tsx

```diff
@@ -107,14 +112,8 @@ function TableHeader<Shape>({
 
   const normalizedAccessor = sortBy && normalizeAccessor(sortBy);
 
-  let glyph: 'unsorted' | 'asc' | 'desc';
-
-  if (sort?.columnId === index) {
-    glyph = sort?.direction ?? 'unsorted';
-  } else {
-    glyph = 'unsorted';
-  }
-
+  const sortDirection = sort && sort.columnId === index ? sort.direction : null;
+  const glyph: 'unsorted' | 'asc' | 'desc' = sortDirection ?? 'unsorted';
```

Comment:
should we create an enum for these values?

Reply:
something we can consider doing if we use these values in more places